__tests__/sportstg.test.js

```diff
@@ -1,9 +1,19 @@
 /**
  * @jest-environment node
  */
+const assert = require('assert') //For AssertionError object definition
 const connector = require('../src/sportstg-api')
 const compId = '0-10486-0-507859-0'
 
+test('Get ladder validates compId', () => {
+    expect.assertions(1)  
+    return connector.getLadder()
+        .then(() => {}) //Should not return ladder object
```

Comment:
Regarding these fail comments, I looked into what others have used to fail a test on a resolved promise, and there are a number of alternatives to what I've used, but nothing explicit such as a `jest.fail()` equivalent. See [this issue](https://github.com/facebook/jest/issues/2129) and [this one](https://github.com/facebook/jest/issues/1377) for examples.

As is these tests fail when the promise resolves, since no assertion is called as per `expect.assertions(1)`, but we could just add a `expect(true).toBe(false)` if you'd like for the test to be more explicit. Let me know your thoughts.

Reply:
Actually, I've found this way: https://jestjs.io/docs/en/expect#rejects

```js
test('rejects to octopus', async () => {
  await expect(Promise.reject(new Error('octopus'))).rejects.toThrow('octopus');
});
```
Important note from the documentation:

> If the promise is fulfilled the assertion fails.

So it looks like what we want in one line of ~Perl~ Javascript